examples/buildkit.yaml

```diff
@@ -0,0 +1,30 @@
+# Example to use Buildkit
+# $ limactl start ./buildkit.yaml
+
+# To run `buildkit` on the host (assumes buildctl is installed):
+# $ export BUILDKIT_HOST=unix://$HOME/.lima/buildkit/buildkitd.sock
```

Comment:
To work even if the template is used for a different setup, use a variable instead:

`export BUILDKIT_HOST=$(limactl list buildkit --format 'unix://{{.Dir}}/sock/buildkitd.sock')`

Note that the location needs to match whatever was forwarded below, to work...

Reply:
ah thank you, really appreciate this kind of awesome informations 🤩